### ISSUE TITLE
[Designer]Bug 37671664: [ActivityUpdate.json]: Luminosity ratio of focused placeholder text 'Add a comment' is 3.1:1 with respect to the background, which is less than 4.5:1.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7617]
### Problem Description

[Bug 37671664](https://microsoft.visualstudio.com/OS/_workitems/edit/37671664): [ActivityUpdate.json]: Luminosity ratio of focused placeholder text 'Add a comment' is 3.1:1 with respect to the background, which is less than 4.5:1.

Test Environment:

Application: Adaptive Cards
OS version: Dev (OS Build 22499.1010)
Application Version: 3.0.2111.24001
Tool: Color Contrast Analyzer

Repro steps:

1. Launch the Adaptive Card application.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to ActivityUpdate.Json button by using Tab key and activate it.

4. Navigate to "Add a comment" edit box by using Tab key..

5. Observe the issue with the tool.

Actual Result:
Luminosity ratio of focused placeholder text 'Add a comment' is 3.1:1 with respect to the background, which is less than 4.5:1.

Expected Result:
Luminosity ratio of focused placeholder text 'Add a comment' with respect to the background should be greater or equal to 4.5:1

User Experience:
Low vision users will be impacted if with out keyboard focus luminosity contrast ratio of focused 'Add a comment' placeholder text with respect to background is less than as per expected.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vagpt** [comment]
Now this UI is not appearing hence closing this issue.

![Image](https://github.com/user-attachments/assets/44c9cfc1-7c91-4fa9-b094-b823be8e7388)
